packages/design-system/src/components/Tooltip/TooltipTrigger.stories.tsx

```diff
@@ -19,6 +19,7 @@ import React from "react";
 import { Story, Meta } from "@storybook/react";
 import { TooltipTrigger } from "./TooltipTrigger";
 import { Button } from "../Button";
+import { palette } from "../../styles";
 
 export default {
```

Comment:
if you specify a color control here (see e.g. `Icons.stories.tsx`) Storybook will display a color picker instead of a plain text input

Reply:
Oh yes, thank you!